apps/server/src/modules/files-storage/mapper/scan-result.mapper.ts

```diff
@@ -0,0 +1,22 @@
+import { ScanResultDto, ScanResultParams } from '../controller/dto';
+import { ScanStatus } from '../entity';
+
+export function mapScanResultParamsToDto(scanResultParams: ScanResultParams): ScanResultDto {
+	const scanResult = new ScanResultDto({
+		status: ScanStatus.VERIFIED,
+		reason: 'Clean',
+	});
+
+	if (scanResultParams.virus_detected) {
+		scanResult.status = ScanStatus.BLOCKED;
+		scanResult.reason = scanResultParams.virus_signature ?? 'Virus detected';
+	} else if (scanResultParams.error) {
+		scanResult.status = ScanStatus.WONT_CHECK;
+		scanResult.reason = scanResultParams.error;
+	} else if (scanResultParams.virus_detected === undefined) {
```

Comment:
@SevenWaysDP Diesen Fall habe ich in den Tests von der alten Methode gefunden. Ist der noch gültig?

Reply:
Nein bitte entfernen sonst haben wir wieder ein Problem. In alle anderen Fälle soll Default definierte scanResult zurück kommen.